.github/workflows/build.yml

```diff
@@ -437,6 +437,11 @@ jobs:
           cat >> _update_spec << EOF
           # This file is used for regulating self-update
           lock 2022.08.18.36 .+ Python 3.6
+          lock 2023.11.14 .+ Python 3.7
+          lockV2 yt-dlp/yt-dlp 2022.08.18.36 .+ Python 3.6
```

Comment:
is pointless, no?
```suggestion
```

No harm in keeping it though

Reply:
yeah it should be pretty much impossible for a py3.6 user to have an updater that processes lockV2, but I opted for caution